x/ibc/02-client/client/cli/query.go

```diff
@@ -41,7 +41,8 @@ func GetCmdQueryClientStates() *cobra.Command {
 			}
 
 			clientCtx = clientCtx.WithHeight(height)
-			return clientCtx.PrintOutput(clientStates)
+
+			return clientCtx.PrintOutputLegacy(clientStates)
```

Comment:
there are all proto messages, I assume we can use `PrintOutput`

Reply:
It's not that simple, you're returning an array of interfaces. jsonpb doesn't know what to do with that. I'd prefer if you guys figure out how you want to approach that.

Reply:
see #6987